crates/environ/src/fact/trampoline.rs

```diff
@@ -39,6 +40,9 @@ use wasmtime_component_util::{DiscriminantSize, FlagsSize};
 const MAX_STRING_BYTE_LENGTH: u32 = 1 << 31;
 const UTF16_TAG: u32 = 1 << 31;
 
+const EXIT_FLAG_ASYNC_CALLER: i32 = 1 << 0;
+const EXIT_FLAG_ASYNC_CALLEE: i32 = 1 << 1;
```

Comment:
These constants seem like they might be used by the host runtime as well, is that right? If so should they be `pub` and somewhere accessible from the outside of this crate?